docs/reference/deploy/master-config-reference.rst

```diff
@@ -1439,6 +1439,12 @@ Whether this feature is enabled. Defaults to ``true``.
 Integer identifier of a role to be assigned. Defaults to ``2``, which is the role id of
 ``WorkspaceAdmin`` role.
 
+``initial_user_password``
+=========================
+
+Initial password for the built-in ``determined`` and ``admin`` users. Applies on first launch when a
+cluster's database is bootstrapped, otherwise it is ignored.
+
```

Comment:
should we tell them it forces an initial password?

Reply:
This configuration does not force an initial user password -- I didn't go that far yet. Just the deploy tool for cloud clusters does.